RecoLocalCalo/HcalRecProducers/src/HcalSimpleReconstructor.cc

```diff
@@ -59,17 +47,6 @@ HcalSimpleReconstructor::HcalSimpleReconstructor(edm::ParameterSet const& conf):
   
 }
 
-void HcalSimpleReconstructor::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
-  edm::ParameterSetDescription desc;
```

Comment:
if HcalSimpleReconstructor is still used by something, the fillDescriptions method should stay with a default matching an existing use case (or even all existing use cases).